Dark mode support (#1913)
jazzband/django-debug-toolbar: docs/changes.rst

* Added check to prevent the toolbar from being installed when tests
  are running.
* Added test to example app and command to run the example app's tests.
* Implemented dark mode theme and button to toggle the theme,

>   introduced config var ``DEFAULT_THEME`` to set default theme to use.

Reviewer: ```suggestion
  introduced the ``DEFAULT_THEME`` setting which sets the default theme to use.
```

The spell checker doesn't like `config var`.
